library/psa_crypto_pake.c

```diff
@@ -359,22 +388,18 @@ static psa_status_t psa_pake_ecjpake_setup( psa_pake_operation_t *operation )
     else
         return( PSA_ERROR_BAD_STATE );
 
-    if( psa_is_valid_key_id( operation->password, 1 ) == 0 )
+    if (operation->password == NULL ||
+        operation->password_len == 0 )
```

Comment:
Minor: either of these checks is enough on its own (the pointer is null iff the length is 0). The code is correct but weird.